src/main/java/org/carlmontrobotics/Constants.java

```diff
@@ -77,6 +77,9 @@ public static final class Effectorc {
 		public static final double PODIUM_RPM = 4000;
 		public static final double SPEAKER_RPM = 2100;
 		public static final double[] RPM_SELECTOR = {AMP_RPM, SUBWOOFER_RPM, PODIUM_RPM};
+		public static final int AMP_RPM_INDEX = 0;
+		public static final int SUBWOOFER_RPM_INDEX = 1;
+		public static final int PODIUM_RPM_INDEX = 2;
```

Comment:
IMO its really annoying to have to add an index everytime we want a new arm position. This feels like something only the person who originally wrote the code would know to do.